src/components/formSections/components/TermsListField.js

```diff
@@ -120,9 +119,29 @@ export default class TermsListField extends React.Component {
     );
   }
 
+  validateNoteField = (values, termValue) => {
+    const val = values ? values[termValue] : [];
+    const { note, value } = val ? val[0] : {};
+    if (note && !value) {
+      return {
```

Comment:
Where are you using this return value? Is this another refactoring artifact that's been left over from a previous iteration? The handler prop looks like it just sets a boolean.